Assignment-3/app/src/components/LogInAgain.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const LogIn = () => {
+  return (
+    <div>
+        
+      
+    </div>
+  )
+}
```

Comment:
Why do we have this component

Reply:
no use of this componnet so i can remove it